src/platform/plugins/shared/console/public/application/containers/editor/styles.ts

```diff
@@ -20,18 +20,13 @@ export const useActionStyles = () => {
       top: 0;
       // Adjust for possible scrollbars
       right: ${euiTheme.size.base};
-      height: ${euiTheme.size.l};
-      background-color: ${euiTheme.colors.backgroundBaseSubdued};
+      height: calc(${euiTheme.size.l} + ${euiTheme.size.s});
```

Comment:
Why didn't you use `euiTheme.size.xl` here?

`l` = 24px
`s`= 8px
`l + s = 32px = xl`

```suggestion
      height: ${euiTheme.size.xl};
```